src/app/page.tsx

```diff
@@ -1,47 +1,53 @@
-import Image from "next/image";
+"use client";
+
+import { PairsList } from "./PairsList";
+import { createContext, useEffect, useState } from "react";
+import * as adex from "alphadex-sdk-js";
+import { PairInfo } from "./PairInfo";
+
+export const AdexStateContext = createContext(null);
 
 // more components here: https://daisyui.com/components/
 
+adex.init(); //Connect to alphadex websocket
+
 export default function Home() {
+  const [adexState, setAdexState] = useState(adex.clientState);
```

Comment:
change this line to: 
 const [adexState, setAdexState] = useState(initial_staticState);

Similar to above, we are setting the useState variable to a type StaticType so it can be updated later with the stateChanged$ subscription.